easybuild/framework/easyblock.py

```diff
@@ -1065,7 +1065,7 @@ def make_module_req_guess(self):
             'LIBRARY_PATH': ['lib', 'lib64', 'lib32'],
             'CPATH': ['include'],
             'MANPATH': ['man', 'share/man'],
-            'PKG_CONFIG_PATH': ['lib/pkgconfig', 'share/pkgconfig'],
+            'PKG_CONFIG_PATH': ['%s/pkgconfig' % x for x in ['lib', 'lib64', 'lib32', 'share']],
```

Comment:
@boegel Why alphabetic? then should not be `'LIBRARY_PATH': ['lib', 'lib64', 'lib32']` also alphabetic?
And os.path.join for manpaht as well?